install/kubernetes/cilium/templates/cilium-etcd-operator-deployment.yaml

```diff
@@ -67,6 +67,14 @@ spec:
         command:
         - /usr/bin/cilium-etcd-operator
         env:
+        {{- if and (.Values.k8sServiceHost) (.Values.k8sServicePort) }}
+        - name: KUBERNETES_SERVICE_PORT
+          value: "{{ .Values.k8sServicePort }}"
+        - name: KUBERNETES_SERVICE_HOST
+          value: {{ .Values.k8sServiceHost }}
+        - name: KUBERNETES_Port
```

Comment:
this should be `KUBERNETES_PORT`? Also, I don't think this env variable is needed it

Reply:
Sure I can try dropping it.